src/services/locationSearchService.js

```diff
@@ -1,27 +1,37 @@
+import config from "../config";
+
 import Constants from "../utils/Constants";
 
+const { API_HOSTNAME } = config;
+
 export const searchCities = async (prefix, limit = 10) => {
 
-    const API_ENDPOINT = "https://wft-geo-db.p.rapidapi.com/v1/geo/cities";
+    // const API_ENDPOINT = "https://wft-geo-db.p.rapidapi.com/v1/geo/cities";
     const options = {
-        types: "CITY",
-        minPopulation: "100000",
-        namePrefix: prefix,
+        // types: "CITY",
+        // minPopulation: "100000",
+        // namePrefix: prefix,
         limit,
+        searchTerm: encodeURI(prefix),
     };
+
     try {
-        const res = await fetch(`${API_ENDPOINT}?${Object.entries(options).map(([k, v]) => `${k}=${v}`).join("&")}`, {
+        const res = await fetch(`${API_HOSTNAME}/location/search?${Object.entries(options).map(([k, v]) => `${k}=${v}`).join("&")}`, {
             method: "GET",
             headers: {
-                "x-rapidapi-host": "wft-geo-db.p.rapidapi.com",
-                "x-rapidapi-key": process.env.GEO_API_KEY,
+                // "x-rapidapi-host": "wft-geo-db.p.rapidapi.com",
+                // "x-rapidapi-key": process.env.GEO_API_KEY,
             },
         });
 
         if (!res.ok) return [];
 
-        const data = (await res.json()).data;
-        return data.map(({ city, country, latitude, longitude }) => ({ city, country, latitude, longitude }));
+        // const data = (await res.json()).data;
+        // return data.map(({ city, country, latitude, longitude }) => ({ city, country, latitude, longitude }));
+
+        const data = await res.json();
+
+        return data;
 
     } catch (err) {
         throw [{ msg: Constants.UNEXPECTED_ERROR_MESSAGE }];
```

Comment:
Maybe you should not modify this file in this PR

Reply:
I don't now how this got through...